packages/runtime/src/next/api-handler/handlers/manifest.ts

```diff
@@ -14,6 +14,7 @@ export type Manifest = {
   siteVersions: boolean
   unstable_siteVersions: boolean
   localizedPageSSR: boolean
+  webhook: boolean
```

Comment:
>What if the manifest provided the URL? 🤔

@migueloller We can make this dynamic and return the string value depending on where user put the api handler. 

But I think there's little value in doing that right now, given that we also hardcoded the `manifest` URL and other endpoints. I feel like it's more important to make the `manifest` URL configurable first. But then it's in the realm of the first-class host project.

Reply:
I second this, at least for now

Reply:
That sounds good to me. We have plans to rework this with first-class hosts anyway